git_machete/cmd.py

```diff
@@ -98,6 +98,8 @@ def read_definition_file(self, verify_branches: bool = True) -> None:
 
         invalid_branches: List[str] = []
         for index, line in enumerate(lines):
+            if line == "" or line.isspace():
```

Comment:
Whoops... so the previous impl did NOT sift out the lines that were **empty**?

Reply:
Generally! if you could add some regression test(s) that would fail on the previous impl but pass on the new one

Reply:
Previous impl did remove empty lines but before enumerate, so info about line number was lost in case there was empty line in .git-machete.

Ok, i'll add regression test for this issue.